index.html

```diff
@@ -68,6 +68,8 @@
          * it */
         if (target) {
             var qr = new QRCode(target, {
+                width : 500,
```

Comment:
The problem is here: without hardcoded pixels the qrcode looks really blurry on my presentations. I tried to make use of useSVG instead, but SVG is buggy in qrcode (needs a fix in the project) and I didn't manage to get the link on the page with inline svg for some reasons. I could try again later to investigate further.